applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_solver_wrapper.py

```diff
@@ -160,6 +160,8 @@ def _GetDataCommunicator(self):
     def _GetDefaultParameters(cls):
         return KM.Parameters("""{
             "type"                    : "",
+            "analysis_module"         : "",
```

Comment:
This field already exist in the current `ProjectParameters.json` and we call it `analysis_stage`. I suggest to be consistent with that in here.

Reply:
Done